javascript/chronometer.js

```diff
@@ -1,26 +1,39 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
+
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+    this.currentTime += 1
+  }, 1000);
+  }
+
+    getMinutes() {
+      let min = Math.floor(this.currentTime / 60);
+      return min;
+    }
+
+    getSeconds() {
+      let min = Math.floor(this.currentTime / 60);
+      let sec = this.currentTime - min * 60;
+      return sec;
+    }
+
+    twoDigitsNumber(digit) {
+
+       if (digit < 10) { 
+       return "0" + digit
+    }
```

Comment:
this condition will work only if the number has only one digit, but will not do anything if the number has two digits. The test will pass but you need to consider this option as well and just return the digit unchanged in that case.